batch/Makefile

```diff
@@ -44,27 +41,27 @@ push-test: build-test
 run-docker: build
 	docker run -e BATCH_USE_KUBE_CONFIG=1 -i -v $(HOME)/.kube:/root/.kube -p 5000:5000 -t batch
 
-run: build/conda-env
-	. ../loadconda && conda activate hail-batch && python -c 'import batch.server; batch.server.serve('$(BATCH_PORT)')'
+run:
+	python3 -c 'import batch.server; batch.server.serve('$(BATCH_PORT)')'
 
 run-local: run
-	. ../loadconda && conda activate hail-batch && BATCH_USE_KUBE_CONFIG=1 python -c 'import batch.server; batch.server.serve('$(BATCH_PORT)')'
+	BATCH_USE_KUBE_CONFIG=1 python3 -c 'import batch.server; batch.server.serve('$(BATCH_PORT)')'
 
 test: push-test
 	sed -e "s,@image@,$$(cat batch-test-image)," \
 	  < test-batch-pod.yaml.in > test-batch-pod.yaml
 	kubectl create -f test-batch-pod.yaml
 
 ifneq ($(PY_CHECK),false)
-test-local: $(PY_CHECKERS)
+test-local: check
 endif
-test-local: build/conda-env
-	. ../loadconda && conda activate hail-batch && POD_NAMESPACE='test' BATCH_USE_KUBE_CONFIG=1 ./test-locally.sh
+test-local:
+	POD_NAMESPACE='test' BATCH_USE_KUBE_CONFIG=1 ./test-locally.sh
 
 # local means server and test client are two processes on one machine
 # in-cluster means in a k8s pod (from which we get k8s creds)
-test-local-in-cluster: build/conda-env $(PY_CHECKERS)
-	. ../loadconda && conda activate hail-batch && POD_NAMESPACE='test' ./test-locally.sh
+test-local-in-cluster: check
+	POD_NAMESPACE='test' ./test-locally.sh
 
 deploy: push
```

Comment:
concretely, I'm suggesting that either `deploy` does a `docker push batch gcr.io/$(PROJECT)/batch:latest`